packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -539,6 +539,16 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
             },
         });
 
+        const backCompatRuntimeOptions = getBackCompatRuntimeOptions(runtimeOptions);
+        const defaultRuntimeOptions: Required<IContainerRuntimeOptions> = {
+            summaryOptions: { generateSummaries: true },
+            gcOptions: {},
+        };
+        const combinedRuntimeOptions = { ...defaultRuntimeOptions, ...backCompatRuntimeOptions };
+        // if isolated channels are disabled, then there are no .channel layers, we pack at level 1,
```

Comment:
Add that we pack at data store level only. if isolated channels are disabled, then there are no .channel layers, so data stores at level 1 else at level2